src/handlers/cors.js

```diff
@@ -1,17 +1,79 @@
-module.exports = function corsHandler({ allowedOrigins = ['*'] }) {
+module.exports = function corsHandler(
+  {
+    allowedOrigins = ['*'],
+    allowedMethods = ['GET','PUT','POST','PATCH','DELETE','HEAD','OPTIONS'],
+    allowCredentials = true,
+    allowedHeaders = ['Content-Type'],
+    allowedExposeHeaders = ['WWW-Authenticate', 'Server-Authorization'],
+    maxAge = '600',
```

Comment:
Should the maxAge be a string or an int? Looks like it's stringified later so maybe we can keep it as an int here?

Reply:
I meant to allow both so if someone sets this as either one it will just work. According to your last comment it will anyway allow both. I will change this to int.

Reply:
Thanks!